Src/Support/Google.Apis.Auth/GoogleJsonWebSignature.cs

```diff
@@ -55,7 +57,27 @@ public class GoogleJsonWebSignature
         /// <returns>The JWT payload, if the JWT is valid. Throws an <see cref="InvalidJwtException"/> otherwise.</returns>
         /// <exception cref="InvalidJwtException">Thrown when passed a JWT that is not a valid JWT signed by Google.</exception>
         public static Task<Payload> ValidateAsync(string jwt, IClock clock = null, bool forceGoogleCertRefresh = false) =>
-            ValidateAsync(jwt, new ValidationSettings { Clock = clock, ForceGoogleCertRefresh = forceGoogleCertRefresh });
+            ValidateWithCancellationAsync(jwt, clock, forceGoogleCertRefresh);
+
+        /// <summary>
+        /// Validates a Google-issued Json Web Token (JWT).
+        /// Will throw a <see cref="InvalidJwtException"/> if the passed value is not valid JWT signed by Google.
+        /// </summary>
+        /// <remarks>
+        /// <para>Follows the procedure to
+        /// <see href="https://developers.google.com/identity/protocols/OpenIDConnect#validatinganidtoken">validate a JWT ID token</see>.
+        /// </para>
+        /// <para>Google certificates are cached, and refreshed once per hour. This can be overridden by setting
+        /// <paramref name="forceGoogleCertRefresh"/> to true.</para>
+        /// </remarks>
+        /// <param name="jwt">The JWT to validate.</param>
+        /// <param name="clock">Optional. The <see cref="IClock"/> to use for JWT expiration verification. Defaults to the system clock.</param>
+        /// <param name="forceGoogleCertRefresh">Optional. If true forces new certificates to be downloaded from Google. Defaults to false.</param>
+        /// <param name="cancellationToken">Cancellation token</param>
+        /// <returns>The JWT payload, if the JWT is valid. Throws an <see cref="InvalidJwtException"/> otherwise.</returns>
+        /// <exception cref="InvalidJwtException">Thrown when passed a JWT that is not a valid JWT signed by Google.</exception>
+        public static Task<Payload> ValidateWithCancellationAsync(string jwt, IClock clock = null, bool forceGoogleCertRefresh = false, CancellationToken cancellationToken = default) =>
```

Comment:
The problem with this naming pattern is that it can explode very quickly. Next option we have to add is then `ValidateWithCancellationAndOptionAsync`, etc.
So, let's not add this method and only add the overload `ValidateAsync(string, ValidationSettings, CancellationToken)` then `ValidateAsync(string, IClock, bool)` can stay for backward compatibility as a very specific shortcut (to not having to create settings).

Reply:
Alright. Removed.